astropy/units/format/generic.py

```diff
@@ -466,6 +455,30 @@ def _parse_unit(cls, s, detailed_exception=True):
         else:
             raise ValueError()
 
+    _unit_symbols = {
+        '%': 'percent',
+        '\N{PRIME}': 'arcmin',
+        '\N{DOUBLE PRIME}': 'arcsec',
+        '\N{MODIFIER LETTER SMALL H}': 'hourangle',
+        'e\N{SUPERSCRIPT MINUS}': 'electron',
+    }
+
+    _prefixable_unit_symbols = {
+        '\N{GREEK CAPITAL LETTER OMEGA}': 'Ohm',
+        '\N{LATIN CAPITAL LETTER A WITH RING ABOVE}': 'Angstrom',
```

Comment:
Does this mean things like `mÅ` are now allowed?

Reply:
Only the prefixes that are allowed with 'Angstrom' generally - which indeed includes `mÅ` (#11788) but nothing else. Of course, we already had `kOhm`, etc.